Packs/PaloAltoNetworks_SecurityAdvisories/Integrations/PaloAltoNetworksSecurityAdvisories/test_data/advisories4.json

```diff
@@ -0,0 +1,395 @@
+                                                    "version_affected": "!>=",
+                                                    "version_value": "6.1.5"
+                                                },
+                                                {
+                                                    "version_name": "6.2",
+                                                    "version_affected": "!>=",
+                                                    "version_value": "6.2.3 (ETA: Week of 11/27)"
+                                                }
+                                            ]
+                                        }
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            },
+            "problemtype": {
+                "problemtype_data": [
+                    {
+                        "description": [
+                            {
+                                "lang": "eng",
+                                "value": "CWE-754 Improper Check for Unusual or Exceptional Conditions"
+                            }
+                        ]
+                    }
+                ]
+            },
+            "description": {
+                "description_data": [
+                    {
+                        "lang": "eng",
+                        "value": "BGP software such as FRRouting FRR included as part of the PAN-OS, Prisma SD-WAN ION, and Prisma Access routing features enable a remote attacker to incorrectly reset network sessions though an invalid BGP update. This issue is applicable only to devices and appliances with BGP routing features enabled.\n\nThis issue requires the remote attacker to control at least one established BGP session that is propagated to the router to exploit it. The denial-of-service (DoS) impact on the network is dependent on the network's architecture and fault tolerant design.\n\nPrisma Access ‘Security Processing Node Endpoint Remote Network (SP-RN/Branches)' and 'Service Connections (SCs/CANs)' nodes do not peer with the Internet and do not receive Internet routes directly unless explicitly configured by the customer. Prisma Access Nodes are commonly protected by unaffected customer-premise equipment (CPE router devices). Hence the impact of this issue on Prisma Access is limited.\n\nFurther details about this issue can be found at: https://blog.benjojo.co.uk/post/bgp-path-attributes-grave-error-handling\n"
+                    }
+                ]
+            },
+            "references": {
+                "reference_data": [
+                    {
+                        "refsource": "CONFIRM",
+                        "url": "https://security.paloaltonetworks.com/CVE-2023-38802",
+                        "name": "https://security.paloaltonetworks.com/CVE-2023-38802"
```

Comment:
Please change it to `https://dummy.com/CVE-2023-11111`,
And also with all the urls and emails in the json file .
email example : `dummy@dummy.com`

Reply:
these are all public sites and public information.  Nothing needs to be sanitized from those files as the data is public

Reply:
You're right, but we have validations that can fall on it, so to avoid future problems, let's change it. 🙏

Reply:
🙏